cmd/snap-confine/ns-support.c

```diff
@@ -288,6 +382,11 @@ void sc_create_or_join_ns_group(struct sc_ns_group *group,
 		debug
 		    ("successfully re-associated the mount namespace with the namespace group %s",
 		     group->name);
+
+		bool should_discard_ns =
+		    should_discard_current_ns(base_snap_dev);
+		debug("should the namespace be discarded: %s",
```

Comment:
Also a very vague message, and curious wording too.

Note that "namespace" will provide zero hints of what this is really about even to developers that understand in depth the ideas we're dealing with here.

Instead this could be something like:
```
if (should_discard_ns) {
    debug("discarding obsolete base filesystem namespace")
}

Reply:
Ah, sorry, this is just a temporary measure before the rest of this feature is present. I'll update this too.